src/style.scss

```diff
@@ -9,4 +9,5 @@
   position: fixed;
   right: 0;
   width: 90vw;
+  z-index: 100;
```

Comment:
if you want this to be at the very top add a way bigger number and add a comment explaining this is to keep it in the top most layer